src/web_interface/static/js/fact_statistics.js

```diff
@@ -70,7 +70,19 @@ function set_links(canvas_id, any_chart, link) {
 
 }
 
-function create_horizontal_bar_chart(canvas_id, chart_data, link, value_percentage_present_flag) {
+function set_links_from_data(canvas_id, chart) {
+
+    document.getElementById(canvas_id).onclick = function(evt){
+        var points = chart.getElementsAtEvent(evt);
+        if (chart.data.datasets[0].links !== undefined) {
+            var key = chart.data.datasets[0].links[points[0]._index];
+            window.location = link.replace("PLACEHOLDER", key);
```

Comment:
Where is `link` set? Is there some implicit inheritance?

Reply:
The links are passed to `BarChart` in `create_horizontal_bar_chart()` as part of (chart_)data (which in turn comes from the filter `data_to_chart_with_value_percentage_pairs`)